README.md

```diff
@@ -418,6 +419,34 @@ python3 DeepSpeech.py --n_hidden 2048 --checkpoint_dir path/to/checkpoint/folder
 
 Note: the released models were trained with `--n_hidden 2048`, so you need to use that same value when initializing from the release models.
 
+### Training with augmentation
+
+Augmentation is a useful technique for better generalization of machine learning models. Thus, a pre-processing pipeline with various augmentation techniques on raw pcm and spectrogram has been implemented and can be used while training the model. 
+
+#### Audio Augmentation
+1. **Std for Gaussian additive noise:** ```data_aug_features_additive [0 - 1]```
+2. **Std for Normal distribution around 1 for multiplicative noise:** ```data_aug_features_multiplicative [0 - 1]``` 
+3. **Std for speeding-up tempo. If std is 0, this augmentation is not performed:** ```augmentation_speed_up_std [0 - 1]``` 
+
+#### Spectrogram Augmentation
+Inspired by Google Paper on [SpecAugment: A Simple Data Augmentation Method for Automatic Speech Recognition]( https://arxiv.org/abs/1904.08779)
+1. **Keep rate of dropout augmentation on a spectrogram (if 1, no dropout will be performed on the spectrogram)**: 
+   * Enable / Disable : ```augmentation_spec_dropout_keeprate [0 - 1]``` 
+
+2. **Whether to use frequency and time masking augmentation:** 
+   * Enable / Disable : ```augmentation_freq_and_time_masking [0, 1]```  
```

Comment:
This flag is a boolean, so it's `--augmentation_freq_and_time_masking` to enable and `--noaugmentation_freq_and_time_masking` to disable.